helpers/sendRequestToDlive.js

```diff
@@ -45,6 +45,7 @@ module.exports = ({ operationName, query, variables }) => {
             RES(body);
           });
           res.on('error', function(e) {
+            console.log("TEST");
```

Comment:
Remove console logs that have been added

Reply:
Have changed the Three Items That you have pointed out, found react-Icons version of simular Icons to the Default Window Icons

Reply:
Added / Change / Quality of Life Notes:
- Fixed SVG to be Inline using React-Icons
- Fixed Bug where Dev Tools didnt have a Pointer Cursor Instead
- Removed console.log from sendRequestToDlive.js
- Dev Button now is titled Dev Tools instead of Dev